image/base/slim.Dockerfile

```diff
@@ -74,6 +74,12 @@ ADD install-jemalloc /tmp/install-jemalloc
 RUN /tmp/install-jemalloc
 
 ADD install-nginx /tmp/install-nginx
+ADD nginx-1.26.1.tar.gz.asc /tmp/nginx-1.26.1.tar.gz.asc
+
+# From https://nginx.org/en/pgp_keys.html and nginx doesn't seem to indicate whose public key was used to sign the package
+# so there is some trial and error required here to figure out the right public key.
+ADD pluknet.key /tmp/nginx_public.key
```

Comment:
So `nginx_signing.key` from that pgp_keys page isn't the key used to sign nginx? 😅

Reply:
It is apparently used to sign the OS packages and not the files from the download page.

Reply:
Oh, this confused me as well.

I guess _pluknet_ is Sergey Kandaurov's handle.  Would it help to name the file accordingly?  Perhaps its purpose would be more obvious.

```
image/base/nginx-dev-pluknet.key
```

or

```
image/base/nginx-sergey-kandaurov.key
```

(The file name has no significance to PGP.)

Three other developer keys are listed on that page.  If it is unclear who will be signing future releases, we could vendor and `gpg --import` all four keys.  On `--verify`, `gpg` will pass if it is able to verify a signature against any key.  In other words, this procedure will remain forwards compatible so long as any one of those four people sign a future release.